package.json

```diff
@@ -1,12 +1,17 @@
 {
+  "name": "js-sdk",
   "private": true,
   "workspaces": [
     "packages/*"
   ],
   "devDependencies": {
-    "lerna": "^3.18.4"
+    "lerna": "^3.19.0"
   },
   "scripts": {
-    "lerna": "lerna"
+    "lerna": "lerna",
+    "postinstall": "./install-all.sh",
+    "ci": "./ci-all.sh",
+    "build": "lerna run build",
+    "test": "lerna run test"
```

Comment:
Good to create the alias. FYI, you can also use `npm run lerna -- run build`, like that you can run any lerna command.

Reply:
Yeah sure but it's longer to type, so I thought you would prefer the convenience of a npm script shortcut.